plugins/wallabag.koplugin/main.lua

```diff
@@ -376,34 +376,39 @@ end
 -- headers: defaults to auth if given nil value, provide all headers necessary if in use
 -- body: empty string if not needed
 -- filepath: downloads the file if provided, returns JSON otherwise
-function Wallabag:callAPI( method, apiurl, headers, body, filepath)
+function Wallabag:callAPI( method, apiurl, headers, body, filepath )
```

Comment:
Same thing here, the convention in the file is using spaces. If the convention of the project is to not use them, is it ok for me to go over the whole file? Should i make another PR for this specific thing? Or is it ok to make another commit right here

Reply:
It improves commit log/git blame clarity to do it separately, so probably better to do it as a follow-up. Or just as two structured commits in this PR if you know how.

Reply:
I made a second commit that removes the extra whitespaces, and replaces the `foo["bar"]` to `foo.bar`